Dockerfile

```diff
@@ -80,7 +80,7 @@ RUN pip install nodeenv
 # Must be done after Python requirements, since nodeenv is installed
 # via pip.
 # The node environment is already 'activated' because its .../bin was put on $PATH.
-RUN nodeenv /edx/app/nodeenv --node=12.11.1 --prebuilt
+RUN nodeenv /edx/app/nodeenv --node=16.15.1 --prebuilt
```

Comment:
unfortunately, using the pip nodeenv (cribbed from other Dockerfiles at edx) we need to specify a complete version. I can try to migrate this to use nvm rather than nodeenv which would give us more flexibility but that may have to come after this PR.

Reply:
Yeah, no strong opinions about this; mainly was just curious :) Honestly don't think the effort to migrate to use `nvm` is worth it considering we rarely touch the frontend UIs inside of edx-enterprise 🤷‍♂️